py/sigver/verify.py

```diff
@@ -3,24 +3,20 @@
 
 import sys
 
-from azure.identity import DefaultAzureCredential
-from azure.keyvault.certificates import CertificateClient
-from azure.keyvault.keys import KeyClient
-from azure.keyvault.keys.crypto import CryptographyClient, SignatureAlgorithm
-
 from sha256tree import sha256sum
+import requests
+import json
+import base64
 
-KEY_VAULT_URL = "https://ghaf-devenv-ca.vault.azure.net/"
-CERTIFICATE_NAME = "."
-
+CERTIFICATE_NAME="INT-Ghaf-Devenv-Common"
+url = "https://ghaf-devenv-microsign-aleksandrtserepo-app.azurewebsites.net/api/verify-signature"
```

Comment:
Is the plan to move this to some better url?

Reply:
Yes, sure. Very good point.

Reply:
Not a matter of this PR, but be sure to document adjusting this URL, as well as whole setup of this function, in ghaf-infra setup instructions.

Reply:
sure. good point.